yarn.lock

```diff
@@ -19161,7 +19161,7 @@ minimist-options@^3.0.1:
     arrify "^1.0.1"
     is-plain-obj "^1.1.0"
 
-minimist@~0.0.1, minimist@0.0.8, minimist@1.1.x, minimist@1.2.0, minimist@~1.2.0, minimist@^1.1.0, minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0, minimist@^1.2.5:
+minimist@0.0.8, minimist@1.1.x, minimist@1.2.0, minimist@^1.1.0, minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0, minimist@^1.2.5, minimist@~0.0.1, minimist@~1.2.0:
```

Comment:
Ugh, is this okay?

Reply:
This happened locally for me as well!

I think this change _should_ have been in #8206 , but for some reason it wasn't. Sometimes Yarn gets into a state where it won't notice there is redundancy in the lockfile - I don't quite understand why.

So I think this is fine 👍